bindings/azure/blobstorage/blobstorage.go

```diff
@@ -299,26 +301,30 @@ func (a *AzureBlobStorage) list(ctx context.Context, req *bindings.InvokeRequest
 	blobs := []*container.BlobItem{}
```

Comment:
Just one small tweak please. On L300 (I can't leave a comment there), please replace:

```go
metadata := map[string]string{}
```

with:

```go
metadata := make(map[string]string, 3)
```

Since we're storing (up to) 3 keys, this improves the allocation a bit

Reply:
Making me change things unrelated to my PR I see :p

Reply:
Not really. Before this was always empty. Now it includes 3 keys, so optimizing memory allocations is not necessarily unrelated :)

Reply:
It was not empty before!